src/Http/Routing/src/Matching/DfaMatcher.cs

```diff
@@ -317,6 +332,18 @@ public sealed override Task MatchAsync(HttpContext httpContext)
         await _selector.SelectAsync(httpContext, candidateSet);
     }
 
+    [InlineArray(CandidateSetStackSize)]
+    private struct CandidateStateArray
+    {
+#pragma warning disable CA1823 // Avoid unused private fields
```

Comment:
are all these warnings just because Roslyn needs tuning for the new InlineArray feature?

Reply:
Ya, there's an issue for this now.